### DISCUSSION TITLE
CLEDEV

### DISCUSSION THREAD

**Discussion by AntDeb98** [cedricp/ddt4all#855] [Q&A]
Hi all !
In ddtplugins/rsat4_reset.py, it's mentionned a 'CLEDEV' in order to reset crash ({u'CLEDEV For reset crash': '13041976'}). Does anyone have a CLEDEV in order to READ crash data and not reset or delete it ? I may have the dataframes to send in order to do that but the CLEDEV is missing...
Thanks a lot !

**Comment by KarelSvo** [comment]
To read data simply select the file RSAT4_ACU_eng_v15_20150511T131328.xml. CLEDEV is the dataname for the crash reset command in this file.

**Reply by AntDeb98** [reply]
Thanks for your answer ! But do you know where I can find those file on the github ?

**Reply by AndyIreland** [reply]
should be in your ecus folder

**Reply by AntDeb98** [reply]
Sorry for the disturb, it was just some database installation problem... thanks all, I close the topic :)
